pkg/api/validation/v1beta1-swagger.json

```diff
@@ -1312,6 +1312,31 @@
      }
     }
    },
+   "v1beta1.ISCSI": {
```

Comment:
I ran ```./hack/update-swagger-spec.sh``` but it generated more stuff than just iSCSI (see below). Should I check in the those generated files? 

http://pastebin.com/LTe0kRgw